javav2/example_code/rekognition/Readme.md

```diff
@@ -1,12 +1,52 @@
 # Amazon Rekognition Java code examples
```

Comment:
Amazon Rekognition code examples for the SDK for Java

Reply:
fixed